tests/features/webtools.feature

```diff
@@ -0,0 +1,44 @@
+@api
+Feature: Webtools as administrator
+  In order to manage Webtools on the website
```

Comment:
This is not the goal the business has. Webtools is just the means, not the goal. What does webtools offer as an added value to the website?